common/antes.ts

```diff
@@ -67,8 +67,11 @@ export function getAnteBets(
     probBefore: p,
     probAfter: p,
     createdTime,
-    isAnte: true,
     fees: noFees,
+    isAnte: true,
+    isLiquidityProvision: false,
```

Comment:
This field is deprecated and not used anywhere, I believe? (Liquidity has its own subcollection now)

I think we can remove it.

Reply:
I will do this in a subsequent PR after checking to see whether any bets with this flag exist.